### PR TITLE
Added ChainInfo types

### PR DESCRIPTION
**Pull request by DaniSomoza**
Added ChainInfo Types:

```

export type ChainInfo = {
  chainName: string;
  chainId: number;
  shortName: string;
  nativeCurrency: string;
};

export type SafeInfo = {
  safeAddress: string;
  chainId: number;
  threshold: number;
  owners: string[];
  chainInfo: ChainInfo;
};

```

### CODE REVIEW

**Comment by DaniSomoza** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mikheevm** [comment]
Should we somehow use types from the gateway? https://github.com/gnosis/safe-react-gateway-sdk/blob/main/src/types/chains.ts#L65-L80 or even pass all the data we get from the gateway

File: packages/safe-apps-sdk/src/types/sdk.ts [hunk 1]
```diff
@@ -1,3 +1,9 @@
+import { NativeCurrency, ChainInfo as _ChainInfo } from '@gnosis.pm/safe-react-gateway-sdk';
+
+export type ChainInfo = Pick<_ChainInfo, 'chainName' | 'chainId' | 'shortName' | 'nativeCurrency'>;
+
+export { NativeCurrency };
```
Comment (mikheevm) [inline packages/safe-apps-sdk/src/types/sdk.ts:5]:
```suggestion
export { NativeCurrency } from '@gnosis.pm/safe-react-gateway-sdk';
```

**Review by mikheevm — APPROVED** [review]

File: packages/safe-apps-sdk/src/types/sdk.ts (hunk 1 above)
Comment (mikheevm) [reply packages/safe-apps-sdk/src/types/sdk.ts:5]: and remove the import at the top :P

Comment (DaniSomoza) [reply packages/safe-apps-sdk/src/types/sdk.ts]: Done Thanks! :D

**Pull request merged**